test/android/readManifest.spec.js

```diff
@@ -0,0 +1,20 @@
+const path = require('path');
+const chai = require('chai');
+const expect = chai.expect;
+const readManifest = require('../../src/config/android/readManifest');
+
+describe('readManifest', () => {
+
+  it('should return manifest content if file persists in the folder', () => {
+    const manifestPath = path.join(
+      'test', 'android', 'fixtures', 'AndroidManifest.xml'
+    );
+
+    expect(readManifest(manifestPath)).to.be.an('object');
```

Comment:
out of curiosity, does `fs` respect CWD automatically? Cause you are not passing process.cwd() to readManifest and that still does not throw :dart: